src/charts/loading/LoadingContainer.js

```diff
@@ -0,0 +1,73 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+export const loadingContainerWrapper = ({data, chart, shouldShowLoadingState}, container) => {
+    if (shouldShowLoadingState) {
+        container = (
+            <LoadingContainer
+                data={data}
+                loadingState={chart.loading()}
```

Comment:
Would it make sense for the wrapper to take the loadingState directly, instead of being passed the chart and then calling `.loading()`?